builtins/web/timers.cpp

```diff
@@ -12,13 +12,30 @@
 #define S_TO_NS(s) ((s) * 1000000000)
 #define NS_TO_MS(ns) ((ns) / 1000000)
 
+namespace {
+
+class TimersMap {
+public:
+  std::map<int32_t, api::AsyncTask*> timer_ids_ = {};
```

Comment:
Given the new meaning, I'd prefer changing the name of this map, too:

```suggestion
  std::map<int32_t, api::AsyncTask*> timers_ = {};
```